source/_integrations/teslemetry.markdown

```diff
@@ -194,3 +194,73 @@ These are the entities available in the Teslemetry integration. Not all entities
 ## Vehicle sleep
 
 Constant API polling will prevent most Model S and Model X vehicles manufactured before 2021 from sleeping, so the Teslemetry integration will stop polling these vehicles for 15 minutes, after 15 minutes of inactivity. You can call the `homeassistant.update_entity` service to force polling the API, which will reset the timer.
+
+## Services
+
+Teslemetry provides various custom services to interact with the Tesla Fleet API directly.
+
+### Navigate to coordinates
+
+`teslemetry.navigation_gps_request`
+
+| Field         | Description                | Example                          |
+|---------------|----------------------------|----------------------------------|
+| device_id     | The vehicle's device_id    | 0d462c0c4c0b064b1a91cdbd1ffcbd31 |
+| gps           | Dictionary of coordinates  |                                  |
+| gps.latitude  | Latitude in degrees        | -27.9699373                      |
+| gps.longitude | Longitude in degrees       | 153.4081865                      |
+| order         | Order for this destination | 1                                |
+
+### Set scheduled charging
+
+`teslemetry.set_scheduled_charging`
+
+| Field     | Description                           | Example                          |
+|-----------|---------------------------------------|----------------------------------|
+| device_id | The vehicle's device_id               | 0d462c0c4c0b064b1a91cdbd1ffcbd31 |
+| enable    | Enable or disable scheduled charging. | true                             |
+| time      | Time to start charging in HH:MM       | 6:00                             |
+
+### Set scheduled departure
+
+`teslemetry.set_scheduled_departure`
+
+| Field                           | Description                               | Example                          |
+|---------------------------------|-------------------------------------------|----------------------------------|
+| device_id                       | The vehicle's device_id                   | 0d462c0c4c0b064b1a91cdbd1ffcbd31 |
+| enable                          | Enable or disable scheduled departure     | true                             |
+| preconditioning_enabled         | Enable preconditioning                    | true                             |
+| preconditioning_weekdays_only   | Enable preconditioning on weekdays only   | false                            |
+| departure_time                  | Time to precondition by (HH:MM)           | 6:00                             |
```

Comment:
I don't understand "time to precondition". Is this the time when preconditioning stops?

Reply:
It's the time you plan to depart, and preconditioning should be completed by this time. It technically won't stop preconditioning until you start driving.